xyz.ayrsd.AudioStellar.yml

```diff
@@ -0,0 +1,319 @@
+    # addons
+      - type: git
+        url: https://github.com/danomatika/ofxMidi.git
+        commit: f9d85fd888ba23cf49207b362e2bcc8cfad352ed
+        dest: addons/ofxMidi
+      - type: git
+        url: https://github.com/macramole/ofxJSON.git
+        commit: 8486eee1c752422a1ffd630656c814b2bb53305f
+        dest: addons/ofxJSON
+      - type: git
+        url: https://github.com/genekogan/ofxConvexHull.git
+        commit: 58a1a403e873189c9fb88d9b44988b11bc580821
+        dest: addons/ofxConvexHull
+      - type: git
+        url: https://github.com/macramole/ofxImGui.git
+        commit: 4d2dc03ae1b4c8138caaee4711c3a9c2c71bb7d3
+        dest: addons/ofxImGui       
+      - type: git
+        url: https://github.com/macramole/ofxAudioFile.git
+        commit: a5ed48298f7fa972f32c35e57e5e6d422f4581c5
+        dest: addons/ofxAudioFile
+      - type: git
+        url: https://github.com/macramole/ofxPDSP.git
+        dest: addons/ofxPDSP
+        commit: a001999a55fe1aa0a755fd8c608334882576f5af
+      - type: git
+        url: https://github.com/macramole/ofxAnn.git
+        dest: addons/ofxAnn
+        commit: 3957fdc4a5f56b5da378e1d92e4704212b98f5f2
+      - type: git
+        url: https://github.com/2bbb/ofxAbletonLink.git
+        dest: addons/ofxAbletonLink
+        commit: 4dd95c68337509d34785d0a65eaa7822213b41d6
+        disable-submodules: false
+
+
+  - name: audiostellar
+    buildsystem: simple
+    build-commands:
+      - mkdir -p bin/data
+      - cp -R assets bin/data/
+      - OF_ROOT="$FLATPAK_DEST/lib/oF" make Release -j$FLATPAK_BUILDER_N_JOBS
+      - mkdir $FLATPAK_DEST/audiostellar
+      - cp -ar bin/* $FLATPAK_DEST/audiostellar
+    
+    post-install:
+      - ln -s ../audiostellar/audiostellar $FLATPAK_DEST/bin/audiostellar
+    cleanup:
+      - src
+    sources:
+      - type: git
+        url: https://gitlab.com/ayrsd/audiostellar.git
+        tag: "1.4"
+        commit: "c86be6ad89720678585121175091508c6f2cfd0b"
+#        branch: units
+        # branch: flatpak-test
+        # dest: apps/myApps/audiostellar
+    
+  - name: audiostellarExtra
+    buildsystem: simple
+    build-commands:
+      - cp ./audiostellar-data-analysis $FLATPAK_DEST/audiostellar/
+      - cp ./audiostellar-onset-detection $FLATPAK_DEST/audiostellar/
+      
+      - install -Dm644 xyz.ayrsd.AudioStellar.appdata.xml $FLATPAK_DEST/share/metainfo/xyz.ayrsd.AudioStellar.appdata.xml
+      - install -Dm644 xyz.ayrsd.AudioStellar.desktop $FLATPAK_DEST/share/applications/xyz.ayrsd.AudioStellar.desktop
+      - install -Dm644 ./icon_128.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/xyz.ayrsd.AudioStellar.png
+    sources:
+      - type: file
+        path: xyz.ayrsd.AudioStellar.appdata.xml
+      - type: file
+        path: xyz.ayrsd.AudioStellar.desktop
+      - type: file
+        path: icon_128.png
+      - type: file
+        path: audiostellar-data-analysis
+      - type: file
+        path: audiostellar-onset-detection
```

Comment:
neither of these files exist.

Reply:
should i upload this files as well ?

Reply:
Ideally they would be found upstream. I don't see here how it would know where to get them.